templates/focus-ios.xliff

```diff
@@ -145,6 +201,26 @@
         <source>Stop</source>
         <note>Accessibility label for the stop button</note>
       </trans-unit>
+      <trans-unit id="Cancel">
+        <source>Cancel</source>
+        <note>Label on button to cancel edits</note>
+      </trans-unit>
+      <trans-unit id="Disabled">
+        <source>Disabled</source>
```

Comment:
The generic "disabled/enabled" here won't work for any of the languages that use gender (such as French, Spanish, German, etc).
Unfortunately we'll have to know each time what those strings refer to (a masculine or feminine word?) in order to properly localize in many of our languages.